catalog/spatial/wfs/1.1.0/spatial-wfs-v1_1_0-source/pom.xml

```diff
@@ -23,6 +23,14 @@
     <packaging>bundle</packaging>
     <name>DDF :: Spatial :: WFS :: 1.1.0 :: Source</name>
     <dependencies>
+        <dependency>
```

Comment:
Most of these are not new dependencies. They are dependencies we were already using but not declaring explicitly.